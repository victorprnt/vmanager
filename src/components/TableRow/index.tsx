import * as S from './styles'

const TableRow = () => (
  <S.Wrapper>
    <td>Info 01</td>
    <td>Info 02</td>
    <td>Inof 03</td>
    <td>Info 04</td>
    <td>Info 05</td>
    <td>Info 06</td>
    <td>Info 07</td>
  </S.Wrapper>
)

export default TableRow
