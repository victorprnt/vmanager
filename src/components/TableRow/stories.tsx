// import { Story, Meta } from '@storybook/react'
import TableRow from '.'

// export default {
//   title: 'TableRow',
//   component: TableRow
// } as Meta

// export const Default: Story = () => <TableRow />
