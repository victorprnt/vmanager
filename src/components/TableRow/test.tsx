import { render, screen } from '@testing-library/react'

import TableRow from '.'

describe('<TableRow />', () => {
  it('should render the heading', () => {
    const { container } = render(<TableRow />)

    expect(screen.getByRole('row')).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
