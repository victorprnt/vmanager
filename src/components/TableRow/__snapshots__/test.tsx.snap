// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableRow /> should render the heading 1`] = `
<tr
  class="sc-bczRLJ"
>
  <td>
    Info 01
  </td>
  <td>
    Info 02
  </td>
  <td>
    Inof 03
  </td>
  <td>
    Info 04
  </td>
  <td>
    Info 05
  </td>
  <td>
    Info 06
  </td>
  <td>
    Info 07
  </td>
</tr>
`;
